src/Intl/Icu/README.md

```diff
@@ -2,7 +2,7 @@ Symfony Polyfill / Intl: ICU
 ============================
 
 This component maps a collection of functions/classes using the
-[`symfony/intl`](https://github.com/symfony/intl) package in the abscense of the
+[`symfony/intl`](https://github.com/symfony/intl) package in the absence of the
```

Comment:
I agree! And unless I'm missing some important meaning, I would also change `maps` => `provides`